coffee_time_firmware/test/boiler/temp_control_test.exs

```diff
@@ -67,6 +67,51 @@ defmodule CoffeeTimeFirmware.Boiler.TempControlTest do
     end
   end
 
+  describe "init_boot phase" do
```

Comment:
I think I want the test case where I show that the boiler fill state doesn't dork with the backoff flow.